From ff3fd37e19f3e5bc29b24b8bff60c2ff9fe1d942 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Thu, 6 May 2021 08:52:20 -0400
Subject: [PATCH] Conda changes

---
 oss_scripts/configure.sh | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/oss_scripts/configure.sh b/oss_scripts/configure.sh
index a90ef6d..5526225 100755
--- a/oss_scripts/configure.sh
+++ b/oss_scripts/configure.sh
@@ -37,11 +37,11 @@ function is_macos() {
 # Remove .bazelrc if it already exist
 [ -e .bazelrc ] && rm .bazelrc
 
-if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
+if [[ $(conda list tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
   echo 'Using installed tensorflow.'
 else
   echo 'Installing tensorflow.'
-  pip install tensorflow==2.5.0rc0
+  conda install tensorflow==2.5.0
 fi
 
 if is_windows; then
@@ -49,12 +49,14 @@ if is_windows; then
   sed -i -e 's/":headers",$/":headers", ":windows_static_link_data",/' third_party/icu/BUILD.bzl
 fi
 
-write_to_bazelrc "build:manylinux2010 --crosstool_top=@org_tensorflow//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.1:toolchain"
+#write_to_bazelrc "build:manylinux2010 --crosstool_top=@org_tensorflow//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.1:toolchain"
 write_to_bazelrc "build --spawn_strategy=standalone"
 write_to_bazelrc "build --strategy=Genrule=standalone"
 write_to_bazelrc "build -c opt"
 write_to_bazelrc "build --define=framework_shared_object=true"
 write_to_bazelrc "build --experimental_repo_remote_exec"
+write_to_bazelrc "build --action_env PYTHON_BIN_PATH=\"$PYTHON\""
+
 
 # Config for Android build.
 write_to_bazelrc "build:android --crosstool_top=//external:android/crosstool"
-- 
2.23.0

