From b8dd2d156c6e5bd12148b5f18e64666675353843 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 18 Dec 2020 02:56:27 -0600
Subject: [PATCH] Build fixed in a conda environment

---
 oss_scripts/configure.sh | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/oss_scripts/configure.sh b/oss_scripts/configure.sh
index 5805df2..d02c5af 100755
--- a/oss_scripts/configure.sh
+++ b/oss_scripts/configure.sh
@@ -37,11 +37,11 @@ function is_macos() {
 # Remove .bazelrc if it already exist
 [ -e .bazelrc ] && rm .bazelrc
 
-if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
+if [[ $(conda list tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
   echo 'Using installed tensorflow.'
 else
   echo 'Installing tensorflow.'
-  pip install tensorflow==2.4.0
+  conda install -y tensorflow==2.4.*
 fi
 
 if is_windows; then
@@ -49,12 +49,14 @@ if is_windows; then
   sed -i -e 's/":headers",$/":headers", ":windows_static_link_data",/' third_party/icu/BUILD.bzl
 fi
 
-write_to_bazelrc "build:manylinux2010 --crosstool_top=@org_tensorflow//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.1:toolchain"
+#write_to_bazelrc "build:manylinux2010 --crosstool_top=@org_tensorflow//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.1:toolchain"
 write_to_bazelrc "build --spawn_strategy=standalone"
 write_to_bazelrc "build --strategy=Genrule=standalone"
 write_to_bazelrc "build -c opt"
 write_to_bazelrc "build --define=framework_shared_object=true"
 write_to_bazelrc "build --experimental_repo_remote_exec"
+write_to_bazelrc "build --action_env PYTHON_BIN_PATH=\"$PYTHON\""
+
 
 # Config for Android build.
 write_to_bazelrc "build:android --crosstool_top=//external:android/crosstool"
-- 
2.18.2

