From d2f5695af5be390de7c5e2afbfb1a35a438523f2 Mon Sep 17 00:00:00 2001
From: Archana Shinde1 <archana.shinde2504@gmail.com>
Date: Fri, 22 Dec 2023 09:07:07 +0000
Subject: [PATCH] Bazel version retrieval

---
 oss_scripts/run_build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oss_scripts/run_build.sh b/oss_scripts/run_build.sh
index 1959ddd..869698a 100755
--- a/oss_scripts/run_build.sh
+++ b/oss_scripts/run_build.sh
@@ -14,7 +14,7 @@ fi
 source oss_scripts/configure.sh
 
 # Verify correct version of Bazel
-installed_bazel_version=$(bazel version | grep label | sed -e 's/.*: //')
+installed_bazel_version=$(bazel version | grep label | sed -e 's/.*: \(.*\)\- (@non.*)/\1/g')
 tf_bazel_version=$(head -n 1 .bazelversion)
 if [ "$installed_bazel_version" != "$tf_bazel_version" ]; then
   echo "Incorrect version of Bazel installed."
-- 
2.40.1

