From 7bcd14a08d92740e725aaf2a9675640432c6566a Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Thu, 13 Jul 2023 12:15:05 +0000
Subject: [PATCH] Bazel version retrieval

---
 oss_scripts/run_build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oss_scripts/run_build.sh b/oss_scripts/run_build.sh
index 1959ddd..0373470 100755
--- a/oss_scripts/run_build.sh
+++ b/oss_scripts/run_build.sh
@@ -14,7 +14,7 @@ fi
 source oss_scripts/configure.sh
 
 # Verify correct version of Bazel
-installed_bazel_version=$(bazel version | grep label | sed -e 's/.*: //')
+installed_bazel_version=$(bazel version | grep label | sed -e 's/.*: \(.*\)\- (@non.*)/\1/g')
 tf_bazel_version=$(head -n 1 .bazelversion)
 if [ "$installed_bazel_version" != "$tf_bazel_version" ]; then
   echo "Incorrect version of Bazel installed."
-- 
2.40.1

