From d03a4bb1c154a429753aaea94ab3b339ef62a05b Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 12 Nov 2021 06:09:03 -0500
Subject: [PATCH] TF Text to use its own protobuf

---
 WORKSPACE | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index 9228f59..507164b 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -15,6 +15,16 @@ http_archive(
     patch_args = ["-p1"],
 )
 
+http_archive(
+    name = "com_google_protobuf",
+    sha256 = "bf0e5070b4b99240183b29df78155eee335885e53a8af8683964579c214ad301",
+    strip_prefix = "protobuf-3.14.0",
+    urls = [
+        "https://storage.googleapis.com/mirror.tensorflow.org/github.com/protocolbuffers/protobuf/archive/v3.14.0.zip",
+        "https://github.com/protocolbuffers/protobuf/archive/v3.14.0.zip",
+    ],
+)
+
 http_archive(
     name = "com_google_sentencepiece",
     strip_prefix = "sentencepiece-1.0.0",
-- 
2.32.0

